scripts/histmakers/mw_lowPU.py

```diff
@@ -10,11 +10,16 @@
 parser.add_argument("--noGenMatchMC", action='store_true', help="Don't use gen match filter for prompt muons with MC samples (note: QCD MC never has it anyway)")
 parser.add_argument("--flavor", type=str, choices=["e", "mu"], help="Flavor (e or mu)", default="mu")
 
+parser = common.set_parser_default(parser, "pt", [34, 25, 1000])
```

Comment:
Do you really want uniform binning up to 1000?

Reply:
Also this has to be optimized: for the lowPU we don't want an upper cut on lepton pT, but currently the unfolding needs these arguments so I put a high number for now.